test/types.js

```diff
@@ -204,4 +204,13 @@ describe('Types', function () {
       [new NSObject init];
     `).should.match(/#"<NSObject: 0x[0-9a-f]+> \(of doom\)"/);
   });
+
+  it('should support "choose()"', function () {
+    cycript.execute(`
+      const a = [[NSColor alloc] init];
+      const b = [[NSColor alloc] init];
+
+      choose("NSColor");
+    `).should.match(/\["<NSColor:\s0x[a-f0-9]+.+?","<NSColor:\s0x[a-f0-9]+.+"]/);
+  });
```

Comment:
Let's also bump `mjolner` in this repo once that PR lands.

Reply:
This should be it.